packages/contracts/test/epoch.test.ts

```diff
@@ -1,19 +1,226 @@
 // @ts-ignore
 import { ethers } from 'hardhat'
 import { expect } from 'chai'
-import { IncrementalMerkleTree, SparseMerkleTree } from '@unirep/utils'
+import {
+    genRandomSalt,
+    hash4,
+    IncrementalMerkleTree,
+    SparseMerkleTree,
+    stringifyBigInts,
+    ZkIdentity,
+} from '@unirep/utils'
 import {
     EPOCH_TREE_DEPTH,
     EPOCH_TREE_ARITY,
     STATE_TREE_DEPTH,
     defaultEpochTreeLeaf,
+    Circuit,
+    SignupProof,
+    AGGREGATE_KEY_COUNT,
+    AggregateEpochKeysProof,
 } from '@unirep/circuits'
 
 import { EPOCH_LENGTH } from '../src'
 import { deployUnirep } from '../deploy'
+import { defaultProver } from '@unirep/circuits/provers/defaultProver'
+
+async function bootstrapUsers(attester, epoch, unirepContract) {
```

Comment:
What do you think of putting this in the `@unirep/utils` package? It could be used for tests and for testnet/mainnet deployments to generate data. It would also prevent duplicate code in contracts/core.

This test data could be helpful for building the explorer and for load testing the system.

cc @CJ-Rose

Reply:
or I think we can maintain a `@unirep/test` package? because I think developers can install `@unirep/test` in `devDependencies` instead of like `@unirep/utils` is one of the `dependencies` in `@unirep/core`

and I think we should also bootstrap user state transitions and many attesters data in many epochs 
so there might be many functions we can build in the package

Reply:
Yes I think it's a good idea